src/Particular.CodeRules/tools/install.ps1

```diff
@@ -1,58 +0,0 @@
-param($installPath, $toolsPath, $package, $project)
-
-if($project.Object.SupportsPackageDependencyResolution)
-{
-    if($project.Object.SupportsPackageDependencyResolution())
-    {
-        # Do not install analyzers via install.ps1, instead let the project system handle it.
```

Comment:
These files shouldn't be required any more, because all our projects are now SDK-style, and therefore handle package dependency resolution.